parachain/pallets/ethereum-beacon-client/src/benchmarking/data.rs

```diff
@@ -8,14 +8,10 @@ use snowbridge_beacon_primitives::{
 };
 use sp_core::U256;
 use sp_std::vec;
-use std::time::{SystemTime, UNIX_EPOCH};
```

Comment:
https://github.com/Snowfork/snowbridge/blob/d8879cc9b4fb65b9829e05f2f88f9339033f4274/.github/workflows/parachain.yml#L79

seems will break if we introduce std in benchmark test

Reply:
Have you tested this? There isn't a `no-default-features` flag for the check command so I would think `std` would be included here?

Reply:
> Have you tested this? There isn't a no-default-features flag for the check command so I would think std would be included here?

yes, I run check command manually as following which I would assume same as what in ci, correct me if I'm wrong
```
➜  snowbridge git:(main) ✗ cargo check --manifest-path parachain/Cargo.toml --workspace --exclude snowbridge --features runtime-benchmarks
...
 Compiling snowbridge-ethereum-beacon-client v0.0.1 (/Users/yangrong/Projects/snowbridge/parachain/pallets/ethereum-beacon-client)
  error[E0433]: failed to resolve: use of undeclared crate or module `std`
    --> /Users/yangrong/Projects/snowbridge/parachain/pallets/ethereum-beacon-client/src/benchmarking/data.rs:11:5
     |
  11 | use std::time::{SystemTime, UNIX_EPOCH};
     |     ^^^ use of undeclared crate or module `std`
```